pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -189,7 +189,11 @@ func init() {
 		if err != nil {
 			return nil, err
 		}
-		return NewOpenStack(cfg)
+		cloud, err := NewOpenStack(cfg)
+		if err != nil {
+			klog.V(1).Infof("New openstack client created failed with config")
```

Comment:
maybe we should print the error as well?

Reply:
this actually comes from following error:

error building controller context: cloud provider could not be initialized:
could not init cloud provider "openstack": Resource not found

The `resource not found` will anyway be printed in the log but it's not helpful to know what
is the error come from... so add the info in this PR helps to find where's error 
because resource not found (the real error) anyway will be printed, we don't need to add here?